lola.html

```diff
@@ -457,13 +461,9 @@ <h3>Fetching Migration Outbox</h3>
         </li>
       </ul>
 
-      <p>
-
-        Open Issue:  Should the source server merge “update” and “delete” activity into its content?   Thus, an updated object appears not as the original, but the final updated object, and deleted objects do not appear.  This is the tip of the iceberg of a tradeoff between significantly different approaches - whether we attempt to copy a content repository, or a changelog including the content.  
-      </p>
 
       <p>
-        If the source server has has ‘likes’ information on objects, it should include that collection in each object as per <a href="https://www.w3.org/TR/activitypub/#likes">https://www.w3.org/TR/activitypub/#likes</a>. If the source server has ‘shares’ information, that is included as per <a href="https://www.w3.org/TR/activitypub/#shares">https://www.w3.org/TR/activitypub/#shares</a>. 
+        If the source server has has ‘likes’ information on objects, it should include that collection in each object as per <a href="https://www.w3.org/TR/activitypub/#likes">https://www.w3.org/TR/activitypub/#likes</a>. If the source server has ‘shares’ information, that is included as per <a href="https://www.w3.org/TR/activitypub/#shares">https://www.w3.org/TR/activitypub/#shares</a>.  The source server MAY limit the size of particularly large likes and shares collections.
```

Comment:
I'm not so sure-- how does the destination server know they got a subset? can the destination server ask for the rest if they know they only got a subset? etc.